python/tvm/meta_schedule/testing/utils.py

```diff
@@ -110,3 +116,46 @@ def initialize_with_tune_context(self, context: "TuneContext") -> None:
 
     def apply(self, trace: Trace, _) -> Optional[Trace]:
         return Trace(trace.insts, {})
+
+
+def apply_fixed_schedules(
+    relay_mod: Union[RelayFunc, IRModule],
+    target: Union[str, Target],
+    params: Optional[Dict[str, NDArray]],
+    schedule_fn: Callable[[ExtractedTask, Schedule], bool],
+):
+    """Apply fixed schedules (manually written, without any tunable knobs) as specified by
+    schedule_fn to extracted tasks, and return a database that can be passed to ApplyHistoryBest.
+
+    Parameters
+    ----------
+    mod : Union[RelayFunc, IRModule]
+        The Relay module to apply fixed schedules.
+    target : Union[str, Target]
+        The target used to extract tasks.
+    params : Optional[Dict[str, tvm.runtime.NDArray]]
+        The associated parameters of the module.
+    schedule_fn : Callable[[ExtractedTask, Schedule], bool]
```

Comment:
What about we change it to `Callable[[ExtractedTask], Schedule]`, i.e., for input the dispatched IRModule is inside of task, and we can return a schedule if it matches our rule, otherwise return None. I think it might be better to avoid confusion.

Reply:
That's possible, but that will force users to write

```
mod = Parse._mod(task.dispatched[0])
sch = Schedule(mod)
```

in every `schedule_fn` callback. I think this boilerplate is non-trivial (users shouldn't care about `dispatched` or `Parse` stuff).

Reply:
That make sense, and I wonder if using the task name and schedule would suffice, if we don't want users to care about details inside of the extracted task.

Reply:
There are some cases where having the relay mod is required. For example, I want to be able to skip tasks based on the output dtype of the compute, which can be retrieved from the relay mod.

Reply:
Sounds good. Thanks for the explaination.